tests/contrib/awards/test_awards_api.py

```diff
@@ -0,0 +1,95 @@
+# -*- coding: utf-8 -*-
+#
+# Copyright (C) 2022 CERN.
+#
+# Invenio-Vocabularies is free software; you can redistribute it and/or
+# modify it under the terms of the MIT License; see LICENSE file for more
+# details.
+
+"""Awards API tests."""
+
+from copy import deepcopy
+from functools import partial
+
+import pytest
+from invenio_indexer.api import RecordIndexer
+from invenio_search import current_search_client
+from jsonschema import ValidationError as SchemaValidationError
+
+from invenio_vocabularies.contrib.awards.api import Award
+
+
+@pytest.fixture()
+def search_get():
+    """Get a document from an index."""
+    return partial(
+        current_search_client.get, Award.index._name, doc_type="_doc"
+    )
+
+
+@pytest.fixture()
+def example_award(db, award_full_data, example_funder):
```

Comment:
moderate: duplicated from conftest? are both needed?

Reply:
I am afraid that, for now, both are needed indeed similarly to funders. 
At this level, we want to create an award programmatically, while in other tests we want to create an award using the service's proxy.